src/Connectors/Consumer/ConnectorFactory.php

```diff
@@ -19,6 +21,6 @@ public static function make(): ConnectorInterface
 
     protected static function requiresPartition(): bool
     {
-        return !is_null(config('kafka.runtime.partition'));
+        return !is_null(Manager::get('partition'));
```

Comment:
it could be something like:

```suggestion
        return Manager::has('partition');
```